node/src/client/mod.rs

```diff
@@ -480,7 +523,7 @@ impl<N: Network, C: ConsensusStorage<N>> NodeInterface<N> for Client<N, C> {
         self.shutdown.store(true, std::sync::atomic::Ordering::Release);
 
         // Abort the tasks.
-        trace!("Shutting down the validator...");
+        trace!("Stopping backgroudn tasks...");
```

Comment:
Please keep this logline unchanged, for better or worse our prerelease tests depend on it. Consider adding a comment above so we don't change it in the future.

Reply:
Should the message say `validator` for the client too? I would at least change that one.

Reply:
Good one, yes can you change it to `Shutting down the client`